### ISSUE TITLE
App no longer opens

### ISSUE THREAD

**Issue by Spit79** [wowawi/support#2]
Have been a paid user of the classic talent calc for a while now.

It was working perfectly.

Since this update it will not load. Please revert it to its old state. I have a load of builds saved in it I hope I have not lost?

This is really a case of “if it aint broke, dont fix”

Thanks!

**Comment by dbarton-uk** [comment]
Thank you for raising this issue, and sorry for the frustration it had caused. 

As long as you don't delete and reinstall the app, then your saved builds should not be affected. 

I will release a patch as soon as I can to aim to resolve the problem, and will check back with you to see if your issue is resolved.

Thanks

**Comment by Spit79** [comment]
FYI.App opens on my iPad as I just installed it fresh but it still does not open on my phone.On my phone the screen goes black like it is launching then fails.BenSent from my iPhoneOn 16 Sep 2023, at 1:05 am, dbarton-uk ***@***.***> wrote:﻿
Thank you for raising this issue, and sorry for the frustration it had caused.
As long as you don't delete and reinstall the app, then your saved builds should not be affected.
I will release a patch as soon as I can to aim to resolve the problem, and will check back with you to see if your issue is resolved.
Thanks

**Comment by Spit79** [comment]
Further to my last after trying the app on the ipad.My feedback is the UI on the new app is not as good as the old one. In the old app a click of the talent showed the tooltip, swiping up or down would allow fine adjustments of the points.The new app is clunky with the tooltips not showing up consistently and having to click all the way up to get back to zero.Please consider reverting to the previous version…the old app was simply perfect once you paid to remove the ads as I had.Thanks, Ben Sent from my iPhoneOn 19 Sep 2023, at 2:47 pm, Ben Swallow ***@***.***> wrote:﻿FYI.App opens on my iPad as I just installed it fresh but it still does not open on my phone.On my phone the screen goes black like it is launching then fails.BenSent from my iPhoneOn 16 Sep 2023, at 1:05 am, dbarton-uk ***@***.***> wrote:﻿
Thank you for raising this issue, and sorry for the frustration it had caused.
As long as you don't delete and reinstall the app, then your saved builds should not be affected.
I will release a patch as soon as I can to aim to resolve the problem, and will check back with you to see if your issue is resolved.
Thanks

**Comment by dbarton-uk** [comment]
Thanks for your follow up and feedback.

I have prepared a fix, which should enable the updated version to start successfully. This patch should be available in the next couple of days. 

I am not planning on reverting back to previous version - I did consider this but it is very tricky to do.

 I will take a look at improving the ui and the point allocation gestures. 
 
Thanks again for your feedback - hoping I can get the app to a point where you prefer it over the previous!
